### PR TITLE
Another b ranch

### PR DESCRIPTION
**Pull request by Stevers1**
Hey review time..

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (chrystalme) [inline .gitignore]: This file should contain files you don't want to commit to your repository. In this project, you are required not to commit `node_modules/` and `test.md`. Kindly add them to this file. You do that by just typing their names, Also for a folder, include the `/` forward slash.

**Review by chrystalme — CHANGES_REQUESTED** [review]
## Hi @Stevers1, 

You have done a good job so far! 👏🏽 👏🏽 
However, there are some issues that you still need to work on to prepare your project for further evaluation
## Highlights 💡 
✔️ Linters is set up properly and all tests are passing
✔️ Github flow is correctly adhered to
✔️ Professional looking README.md file

## Suggested changes ♻️ 
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA). Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (Stevers1) [reply .gitignore]: How do I add them without making a new pull request?

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
-# -Hello-Microverse-
\ No newline at end of file
+# Hello-Microverse
```
Comment (Ismailco) [inline README.md:1]: - [ ] [OPTIONAL] You should use a descriptive description on your PR, kindly change your description so it reflects the changes on the Pull Request

Comment (Ismailco) [inline README.md:1]: - [ ] [OPTIONAL] You should use a descriptive PR title that reflects the changes you made in the PR, kindly implement this

Comment (Ismailco) [inline README.md:1]: - [ ] [OPTIONAL] Kindly consider removing `test.md` file from your repo, this is a requirement but the first reviewer forget to mention it

Comment (Ismailco) [inline README.md:1]: - [ ] [OPTIONAL] The Branch name should be the name of a feature or action, kindly in future projects consider using a meaningful name for your branch

**Review by Ismailco — APPROVED** [review]
### Hi @Stevers1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**